Setup/UpgradeSchema.php

```diff
@@ -69,4 +73,22 @@ private function changeTitleType(SchemaSetupInterface $setup)
             'Demo Title'
         );
     }
+
+    private function addTargetAttribute($setup)
+    {
+        $setup->getConnection()->addColumn(
+            $setup->getTable('snowmenu_node'),
+            'target',
+            [
+                'type' => Table::TYPE_TEXT,
+                'length' => 10,
+                'nullable' => true,
+                'after' => 'title',
+                'default' => '_self',
+                'comment' => 'Link target'
```

Comment:
Missing comma after the last array element

Reply:
done